cdap-examples/WikipediaPipeline/src/main/scala/co/cask/cdap/examples/wikipedia/ClusteringUtils.scala

```diff
@@ -98,15 +98,19 @@ object ClusteringUtils {
    * Store the clustering results in the specified dataset and update accumulators and workflow token.
    */
   def storeResults(sc: SparkContext, sec: SparkExecutionContext,
-                   results: Array[Array[(String, Double)]], tableName: String) = {
+                   results: Array[Array[(String, Double)]], namespace: String, tableName: String) = {
     val numRecords: Accumulator[Int] = sc.accumulator(0, "num.records")
     val initial= new Term("", 0.0)
     val highestScore: Accumulator[Term] = sc.accumulator(initial, "highest.score")(HighestAccumulatorParam)
 
     sec.execute(new TxRunnable {
       override def run(context: DatasetContext) = {
-        val table: Table = context.getDataset(tableName)
```

Comment:
this can just be
```
 table = context.getDataset(namespace, tableName)
```
Since after the changes which I suggested you will not be passing null in namespace